.github/workflows/aws_deploy.yml

```
@@ -17,15 +17,65 @@ permissions:
   contents: write  # This allows the workflow to create tags
 
 jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)